dev/integration_tests/ios_add2app_life_cycle/ios_add2app/MainViewController.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <UIKit/UIKit.h>
+#import <Flutter/Flutter.h>
+
+@interface MainViewController : UIViewController
+
+@property(nonatomic, strong) UIStackView* stackView;
```

Comment:
This isn't used outside the implementation, so move it to the extension.
MainViewController.h
```objective-c
@interface MainViewController : UIViewController
@end
```
MainViewController.m
```objective-c
@interface MainViewController ()
@property(nonatomic, strong) UIStackView* stackView;
@end
```